src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/template.json

```diff
@@ -73,68 +73,64 @@
         {
           "condition": "(NoAuth)",
           "rename": {
-            "Shared/MainLayout.NoAuth.razor": "Shared/MainLayout.razor"
+            "Components/Layout/MainLayout.NoAuth.razor": "Components/Layout/MainLayout.razor"
           },
           "exclude": [
-            "Pages/Authentication.razor",
-            "Shared/LoginDisplay.*.razor",
-            "Shared/MainLayout.Auth.razor",
-            "Shared/RedirectToLogin.razor",
+            "Components/Pages/Authentication.razor",
+            "Components/Login/LoginDisplay.razor",
+            "Components/Layout/MainLayout.Auth.razor",
+            "Components/Login/RedirectToLogin.razor",
             "wwwroot/appsettings.Development.json",
             "wwwroot/appsettings.json"
           ]
         },
         {
           "condition": "(!NoAuth)",
           "rename": {
-            "Shared/MainLayout.Auth.razor": "Shared/MainLayout.razor"
+            "Components/Layout/MainLayout.Auth.razor": "Components/Layout/MainLayout.razor"
           },
           "exclude": [
-            "Shared/MainLayout.NoAuth.razor"
-          ]
-        },
-        {
-          "condition": "(IndividualLocalAuth)",
```

Comment:
LoginDisplay.IndividualMsalAuth.razor and LoginDisplay.IndividualLocalAuth.razor were identical. I replaced them with one LoginDisplay.razor

Reply:
Good catch!